setup.cfg

```diff
@@ -18,11 +18,17 @@ classifier =
 
 [files]
 packages =
+    descarger
```

Comment:
Actually this was wrong from the time I've added shperer. It should be:

```
[files]
packages =
    zuuler

```

Reply:
Oh, good to know :-D Thanks :-) Fixed